src/components/common/connectionDialog/internetConnection.js

```diff
@@ -81,7 +82,7 @@ const InternetConnection = props => {
       //first time that connection is ok, from now on we will start showing the connection dialog on error
       setFirstLoadError(false)
       showDialogWindow && showDialogWindow.cancel()
-      hideDialog()
+      !firstLoadError && hideDialog()
```

Comment:
whats this for?

Reply:
to not hide dialog when it wasnt shown. on firstLoadError the dialog is not shown - so no need to hide.
when hiding dialog without checking for 'forstLoadError' it could close some other dialog not related to the internetConnection. It happend to me with upgrade popup after moving it to RouterSelector